drivers/vulkan/vulkan_context.cpp

```diff
@@ -532,6 +574,35 @@ Error VulkanContext::_create_physical_device() {
 				ERR_FAIL_V(ERR_CANT_CREATE);
 				break;
 		}
+	} else if (enabled_debug_report) {
+		CreateDebugReportCallbackEXT = (PFN_vkCreateDebugReportCallbackEXT)vkGetInstanceProcAddr(inst, "vkCreateDebugReportCallbackEXT");
+		DebugReportMessageEXT = (PFN_vkDebugReportMessageEXT)vkGetInstanceProcAddr(inst, "vkDebugReportMessageEXT");
+		DestroyDebugReportCallbackEXT = (PFN_vkDestroyDebugReportCallbackEXT)vkGetInstanceProcAddr(inst, "vkDestroyDebugReportCallbackEXT");
+
+		if (nullptr == CreateDebugReportCallbackEXT ||
```

Comment:
I think we usually try to avoid multiline ``if`` statements @akien-mga Is this something we enforce?

Reply:
I used clang-format hoping that it will reformat the code in terms of the line width, but now I see that `ColumnLimit` is set to 0, which means that devs decide where to break the line. I will change the if statement to a single line.